kubernetes/loculus/values.yaml

```diff
@@ -10,6 +10,11 @@ disableBackend: false
 disablePreprocessing: false
 disableIngest: false
 disableEnaSubmission: true
+website:
```

Comment:
Hi! Just a thought - would it be possible to make this optional and set the default to true?

Reply:
What do you mean exactly? The defaults of the individual values are `true`?